src/avro2json.c

```diff
@@ -947,6 +979,7 @@ static void print_usage(const char *exe) {
           " --prune            Omit null values as well as empty lists and "
           "objects\n"
           " --logical-types    Convert logical types automatically\n"
+          " --ms-hadoop-logical-types    Convert non-standard logical types of Microsoft.Hadoop.Avro (System.Guid, System.DateTime, System.TimeSpan) automatically\n"
```

Comment:
In this PR you only added support for System.Guid, but not for DateTime and TimeSpan, so the comment is confusing.

Reply:
This flag will serve for all the avro.net logical types. Guid support cannot be added without it.

Reply:
I guess you mean to mention only Guid. Then we need to care not to forget to change this comment when new types are added